edxval/migrations/0004_remove_profile_fields.py

```diff
@@ -1,89 +0,0 @@
-# -*- coding: utf-8 -*-
-from south.utils import datetime_utils as datetime
-from south.db import db
-from south.v2 import SchemaMigration
-from django.db import models
-
-
-class Migration(SchemaMigration):
-
-    def forwards(self, orm):
-        # The extension, width, and height fields of the Profile model are
-        # removed, but we want this migration to be non-destructive. Thus, the
-        # columns are simply altered to allow null values.
-        db.alter_column('edxval_profile', 'width', self.gf('django.db.models.fields.PositiveIntegerField')(null=True, default=1))
-        db.alter_column('edxval_profile', 'extension', self.gf('django.db.models.fields.CharField')(max_length=10, null=True, default='mp4'))
-        db.alter_column('edxval_profile', 'height', self.gf('django.db.models.fields.PositiveIntegerField')(null=True, default=1))
-
-        # On MySQL, attempts to insert new model instances (which lack values
-        # for the removed fields) results in a warning about a lack of a default
-        # value. Because django does not have a mechanism for specifying the
-        # default value in the database, we must manually alter the table here.
-        if db.backend_name == 'mysql':
-            db.execute("ALTER TABLE edxval_profile MODIFY extension VARCHAR(10) DEFAULT 'mp4'")
-            db.execute("ALTER TABLE edxval_profile MODIFY width INT(10) UNSIGNED DEFAULT 1")
-            db.execute("ALTER TABLE edxval_profile MODIFY height INT(10) UNSIGNED DEFAULT 1")
```

Comment:
@nedbat @nasthagiri I have backlogged a ticket to cover this work: https://openedx.atlassian.net/browse/TNL-3292